.github/workflows/playwright.yml

```diff
@@ -104,17 +141,30 @@ jobs:
           cd "$(ls -1)"
           mv ./* "$src/dist/"
 
+      - name: Start Grafana Custom Location
+        if: ${{ inputs.docker-compose-file }}
+        run: |
+          docker compose -f ${{ inputs.docker-compose-file }} pull
+          GRAFANA_VERSION=${{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=${{ matrix.GRAFANA_IMAGE.NAME }} docker compose -f ${{ inputs.docker-compose-file }} up -d
+
       - name: Start Grafana
+        if: ${{ !inputs.docker-compose-file }}
         run: |
           docker compose pull
           GRAFANA_VERSION=${{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=${{ matrix.GRAFANA_IMAGE.NAME }} docker compose up -d
 
       - name: Wait for Grafana to start
         uses: grafana/plugin-actions/wait-for-grafana@main
 
+      - name: Run Dockerized Playwright tests
+        if: ${{ inputs.dockerized-playwright-command }}
+        id: run-tests-in-container
+        run: ${{ inputs.dockerized-playwright-command }}
```

Comment:
Same here, we shouldn't allow running the user input directly. What's an example of the value for this `dockerized-playwright-command` variable?

Reply:
An example would also be an npm script to actually run playwright dockerized:

```
"e2e:ci": " docker run --network host -v $(pwd)/e2e:/app/e2e -v $(pwd)/src:/app/src metrics-drilldown-plugin-e2e:latest"
```